CMakeLists.txt

```diff
@@ -146,18 +146,14 @@ endif(ENABLE_GPS)
 ## Scripting extensions
 option(DISABLE_SCRIPTING "Disables the scripting extensions." OFF)
 if (NOT DISABLE_SCRIPTING)
-createExtension(SCRIPTING-EXTENSIONS "SCRIPTING EXTENSIONS" "This enables scripting" "extensions/script" "${TEST_DIR}/script-tests")
+    createExtension(SCRIPTING-EXTENSIONS "SCRIPTING EXTENSIONS" "This enables scripting" "extensions/script" "${TEST_DIR}/script-tests")
 endif()
 
 ## USB camera extensions
-createExtension(DISABLE_USB_CAMERA
-				USB-CAMERA-EXTENSIONS
-				"USB CAMERA EXTENSIONS"
-				"This enables USB camera support"
-				"extensions/usb-camera"
-				"${TEST_DIR}/usb-camera-tests"
-				"TRUE"
-				"thirdparty/libuvc-0.0.6")
+option(DISABLE_USB_CAMERA "Disables USB camera support." OFF)
+if (NOT DISABLE_USB_CAMERA)
+    createExtension(USB-CAMERA-EXTENSIONS "USB CAMERA EXTENSIONS" "This enables USB camera support" "extensions/usb-camera" "${TEST_DIR}/usb-camera-tests" "TRUE" "thirdparty/libuvc-0.0.6")
```

Comment:
@calebj Thanks for this. Can you change this to if ( ENABLE_USB_CAMERA) This is a feature we should not enable by default as it and other features are already causing issues for those who are building in embedded environments. Since they are a target audience I'd rather they enable features than disable every single one.

In the future we'll move to a slightly different model as an extension to this which makes it easier for them to enable sets of features. That's been the plan since the beginning but we haven't quite gotten there.